mlflow/tracking/client.py

```diff
@@ -255,6 +641,36 @@ def delete_tag(self, run_id, key):
 
         :param run_id: String ID of the run
         :param key: Name of the tag
+
+        .. code-block:: python
+            :caption: Example
+
+            from mlflow.tracking import MlflowClient
+
+            def print_run_info(run):
+                print("run_id: {}".format(run.info.run_id))
+                print("Tags: {}".format(run.data.tags))
+
+            # Create a run with some tags under the default experiment 0
+            client = MlflowClient()
+            tags = {"t": 1, "t2": 2}
```

Comment:
```suggestion
            tags = {"t1": 1, "t2": 2}
```

This is a nit, but I'd use `t1` rather than `t` here.

Reply:
done!